src/spatialdata/_core/spatialdata.py

```diff
@@ -1044,8 +661,7 @@ def write(
                     # reload the image from the Zarr storage so that now the element is lazy loaded,
                     # and most importantly, from the correct storage
                     element_path = Path(self.path) / "images" / name
-                    image = _read_multiscale(element_path, raster_type="image")
-                    self._add_image_in_memory(name=name, image=image, overwrite=True)
+                    _read_multiscale(element_path, raster_type="image")
```

Comment:
I don't get it, why do you do `_read_multiscale(element_path, raster_type="image")` without assigning it to anything? I think it should be `sdata[name] = _read_multiscale(element_path, raster_type="image")`. Same comment for labels, shapes and points.

Reply:
Ah ok I get it, the old code was for reloading lazily those data that could be lazy loaded. Here we should remove those lines of code.